homeassistant/generated/integrations.json

```
@@ -4341,6 +4341,12 @@
       "config_flow": true,
       "iot_class": "local_polling"
     },
+    "overseerr": {
+      "name": "Overseerr",
+      "integration_type": "hub",
```

I _believe_ that this should be a service, also need to set it in the manifest then
```suggestion
      "integration_type": "service",
```